NAM/convnet.cpp

```diff
@@ -95,14 +95,7 @@ void convnet::_Head::process_(const Eigen::MatrixXf& input, Eigen::VectorXf& out
 
 convnet::ConvNet::ConvNet(const int channels, const std::vector<int>& dilations, const bool batchnorm,
                           const std::string activation, std::vector<float>& params, const double expected_sample_rate)
-: ConvNet(TARGET_DSP_LOUDNESS, channels, dilations, batchnorm, activation, params, expected_sample_rate)
-{
-}
-
-convnet::ConvNet::ConvNet(const double loudness, const int channels, const std::vector<int>& dilations,
-                          const bool batchnorm, const std::string activation, std::vector<float>& params,
-                          const double expected_sample_rate)
-: Buffer(loudness, *std::max_element(dilations.begin(), dilations.end()), expected_sample_rate)
+: Buffer(*std::max_element(dilations.begin(), dilations.end()), expected_sample_rate)
```

Comment:
Reverses the two constructors' use of each other; this makes it clearer IMO that the constructor API that includes the loudness does everything the other does plus setting the given loudness.

Similar elsewhere for the other models.